content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -0,0 +1,177 @@
+: The [Datadog API key][1] used to upload the test results.<br/>
+**Default**: `(empty)`
+
+`DD_APPLICATION_KEY` (Required)
+: The [Datadog Application key][2] used to query the tests to be skipped.<br/>
+**Default**: `(empty)`
+
+`DD_SITE` (Required)
+: The [Datadog site][3] to upload results to.<br/>
+**Default**: `datadoghq.com`<br/>
+**Selected site**: {{< region-param key="dd_site" code="true" >}}
+
+`DD_CIVISIBILITY_ITR_ENABLED=true` (Required)
+: Flag to enable intelligent test runner. <br/>
+**Default**: `false`<br/>
+**Note**: Required only during Beta phase
+
+After setting these environment variables, run your tests as you normally do:
+
+{{< tabs >}}
+
+{{% tab "dotnet test" %}}
+
+By using <a href="https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test">dotnet test</a>
+
+{{< code-block lang="bash" >}}
+dd-trace ci run --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{% tab "VSTest.Console" %}}
+
+By using <a href="https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options">VSTest.Console.exe</a>
+
+{{< code-block lang="bash" >}}
+dd-trace ci run --dd-service=my-dotnet-app --dd-env=ci -- VSTest.Console.exe {test_assembly}.dll
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+#### UI activation
+In addition to the environment variables above, the Intelligent Test Runner needs to be activated in [Test Service Settings][4].
+
+### Swift
+
+To enable Intelligent Test Runner, the version of the `dd-sdk-swift` framework must be >= 2.2.0-rc.1. `Code Coverage` option must also be enabled in the test settings of your scheme or testplan, or  `--enable-code-coverage` must be added to your `swift test` command (if using a SPM target).
+
+The following environment variables must also be set:
+
+`DD_TEST_RUNNER`
+: Enables or disables the instrumentation of tests. Set this value to `$(DD_TEST_RUNNER)` so you can enable and disable test instrumentation with a environment variable defined outside of the test process (for example, in the CI build).<br/>
+**Default**: `false`<br/>
+**Recommended**: `$(DD_TEST_RUNNER)`
+
+`DD_API_KEY` (Required)
+: The [Datadog API key][1] used to upload the test results.<br/>
+**Default**: `(empty)`
+
+`DD_APPLICATION_KEY` (Required)
+: The [Datadog Application key][2] used to query the tests to be skipped.<br/>
+**Default**: `(empty)`
+
+`DD_SITE` (Required)
+: The [Datadog site][3] to upload results to.<br/>
+**Default**: `datadoghq.com`<br/>
+**Selected site**: {{< region-param key="dd_site" code="true" >}}
+
+#### UI activation
+In addition to the environment variables above, the Intelligent Test Runner needs to be activated in [Test Service Settings][4].
+
+## Configuration
+
+The default branch is automatically excluded from having Intelligent Test Runner enabled to reduce irrelevant test noise, but it remains configurable. Because impacted tests could be missed by this exclusion, Datadog recommends you include your default branch.
```

Comment:
```suggestion
The default branch is automatically excluded from having Intelligent Test Runner enabled to reduce irrelevant test noise, but it remains configurable. Because impacted tests could be missed by this exclusion, Datadog recommends including your default branch.
```